primitives/core/src/error.rs

```diff
@@ -66,9 +66,6 @@ pub enum ErrorDetail {
 	// of verification data
 	#[codec(index = 10)]
 	VerifyWeb3SignatureFailed,
-	// error when trying to build vc but no eligible identity is found
-	#[codec(index = 11)]
-	NoEligibleIdentity,
```

Comment:
In fact, if we really want to retain the maximal compatibility, we could leave this enum unchanged (so does `definitions.ts`)

So this will just become an error type that will never be emitted.
Good thing is you don't have to change anything in SDK. @jonalvarezz @higherordertech

Reply:
I'm fine with either way - they both have arguments

Will leave it to you to decide